packages/http/src/validator/deserializers/style/form.ts

```diff
@@ -9,14 +10,21 @@ export class FormStyleDeserializer implements IHttpQueryParamStyleDeserializer {
     return style === HttpParamStyles.Form;
   }
 
-  public deserialize(name: string, parameters: IHttpNameValues, schema: ISchema, explode: boolean = true) {
-    const { type } = schema;
+  public deserialize(
+    name: string,
+    parameters: IHttpNameValues,
+    schema?: JSONSchema4 | JSONSchema6 | JSONSchema7,
```

Comment:
We seem to creating this list in a lot of places. Can we make an alias that we control, some sort of list of "supported versions" or something that we can pass around without needing to hint this piped list?

Reply:
Ok done